Calm dataset

pengbo807 opened the pull request:
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

liushz commented:
<img width="1046" alt="image" src="https://github.com/open-compass/opencompass/assets/28834990/b85a2f33-7c8a-46a6-8507-203694d6f7aa">
目前跑出来的结果展示有点冗长，可以考虑添加summarizer来展示重点内容

pengbo807 commented:
Thank you for your feedback. I've updated the pull request to include the summarizer functionality as suggested. Specifically:

- Added a summarizer option to provide concise output with average information for all tasks
- The summarizer can be activated using the `--summarizer calm` flag
- Without this flag, the script will continue to provide detailed task-specific information

Please review the updates and let me know if any further modifications are needed.

liushz reviewed (approved):
LGTM

bittersweet1999 commented:
we revert this PR because of the lint problem, please reopen this PR
